README.md

```diff
@@ -31,10 +31,14 @@ make services-invoke-publish
 ## Dev setup
 
 ```sh
+# Make sure Ruby is installed
```

Comment:
Could replace that with a node server but I didn't know how to do it in one line off the top of my head